### DISCUSSION TITLE
extraction swaggers

### DISCUSSION THREAD

**Discussion by ericjacq92** [ggrebert/arcep-poc-tmf#11] [General]
Pouvez vous extraire les swaggers au format yaml pour qu'on puisse les consulter sans lancer l'application

**Comment by ggrebert** [comment]
Bonjour,

J'ai préparé 2 repos GIT pour le hackathon de la semaine prochaine:

* [common](https://github.com/ggrebert/before-interop-common)

   Ce repo contient la partie Data Model + les APIs outils.

   Le swagger des outils est dispo ici: <https://ggrebert.github.io/before-interop-common/>

* [malfacon](https://github.com/ggrebert/malfacon)

   Ce repo est une implémentation de Malfacon utilisant les outils et data models de common
   
   Le swagger est dispo ici: <https://ggrebert.github.io/malfacon/>


Si tu veux, je suis dispo dans la journée à partir de 10h30, on peut faire une réu sur Teams.
mon adresse orange: geoffrey.grebert@orange.com
